test/units/TEST-04-JOURNAL.journal-gatewayd.sh

```diff
@@ -67,6 +67,21 @@ curl -LSfs \
      --header "Range: entries=$BOOT_CURSOR:5:10" \
      http://localhost:19531/entries >"$LOG_FILE"
 jq -se "length == 10" "$LOG_FILE"
+# Check that follow with no num_entries follows "indefinitely"
+(
+    set +e; \
+    timeout 5 curl -LSfs \
```

Comment:
FTR, I'm aware that this inherently increases duration of the test suite by 5 seconds. However, until this point there wasn't any other check that the `follow` option works and I don't have better idea how to check that the gateway keeps streaming at this level of tests (any other idea is welcome). As the suite already takes several minutes, 5 seconds could be acceptable. It should be also long enough to get the last couple of entries, not killing the curl before all available log lines are returned.